Makefile

```diff
@@ -15,7 +17,13 @@ build: ## Build docker image
 bats: ## Install bats in the project itself
 	@git clone https://github.com/sstephenson/bats.git
 
-prepare-test: bats ## Prepare the bats dependencies
+venv: requirements-dev.txt
```

Comment:
I'm running this locally, but somehow when running this goal, the dependencies are failing with some issues related to this platform is not supported, in my case I'm using MacOS

```
  xcrun -sdk macosx clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -iwithsysroot/System/Library/Frameworks/System.framework/PrivateHeaders -iwithsysroot/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/Headers -arch arm64 -arch x86_64 -I/Users/vmartinez/work/src/github.com/elastic/opbeans-loadgen/venv/include -I/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/include/python3.8 -I/private/var/folders/dg/wfl7w2g90vz9j9fd_56858jh0000gn/T/pip-install-5dhdv67e/uvloop/vendor/libuv/include -c uvloop/loop.c -o build/temp.macosx-10.14.6-x86_64-3.8/uvloop/loop.o -O2
  In file included from uvloop/loop.c:20:
  In file included from /Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/include/python3.8/Python.h:11:
  In file included from /Library/Developer/CommandLineTools/usr/lib/clang/12.0.0/include/limits.h:21:
  In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/limits.h:63:
  /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/sys/cdefs.h:807:2: error: Unsupported architecture
```

See the log output -> https://gist.github.com/v1v/1486c9ebd87f2fb6ff6596de182d8572

Reply:
Looks like you are on 3.8.  Try Python 3.6 if you're on OS X.